### PR TITLE
Fix some cli commands

### PR DESCRIPTION
**Pull request by schustmi**
## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Describe changes
- Fix some cli commands

### CODE REVIEW

File: src/zenml/integrations/sklearn/__init__.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ class SklearnIntegration(Integration):
     """Definition of sklearn integration for ZenML."""
 
     NAME = SKLEARN
-    REQUIREMENTS = ["sklearn==0.24.2"]
+    REQUIREMENTS = ["scikit-learn==1.0.1"]
```
Comment (htahir1) [inline src/zenml/integrations/sklearn/__init__.py:23]: Shouldnt this be >=1.0.1? Lets make the requirements looser

File: src/zenml/io/fileio.py [hunk 2]
```diff
@@ -46,27 +46,17 @@ def copy(src: PathType, dst: PathType, overwrite: bool = False) -> None:
...
-    """Returns true if file exists at path.
-
-    Args:
-        path: Local path in filesystem.
-
-    Returns:
-        True if file exists, else False.
-    """
+def file_exists(path: PathType) -> bool:
+    """Returns `True` if the given path exists."""
```
Comment (htahir1) [inline src/zenml/io/fileio.py:59]: You would take out args and returns in the docstrings?

File: src/zenml/cli/orchestrator.py [hunk 3]
```diff
@@ -25,6 +25,7 @@
     from zenml.orchestrators.base_orchestrator import BaseOrchestrator
 
 
+@cli_utils.activate_integrations
```
Comment (htahir1) [inline src/zenml/cli/orchestrator.py:28]: This is a very good solution

**Review by htahir1 — COMMENTED** [review]
Done! Just a few comments and a suggestion

File: src/zenml/io/fileio.py (hunk 2 above)
Comment (schustmi) [reply src/zenml/io/fileio.py:59]: For such super small ones I'd say yes, seems like it's kind of self-explanatory. But can also add it back in if you think that makes sense

File: src/zenml/integrations/sklearn/__init__.py (hunk 1 above)
Comment (schustmi) [reply src/zenml/integrations/sklearn/__init__.py:23]: Agreed, I'll remove the version requirement completely for now, similar to the other integrations (I'll also add airflow==2.2.0, as in this case it actually seems to be only working with that version)

**Pull request merged**